src/main/java/com/faforever/client/chat/UserInfoWindowController.java

```diff
@@ -443,7 +443,7 @@ public String toString(Number object) {
         int number = object.intValue();
         int numberOfDataPoints = dataPoints.size();
         int dataPointIndex = number >= numberOfDataPoints ? numberOfDataPoints - 1 : number;
-        return DATE_FORMATTER.format(dataPoints.get(dataPointIndex).getInstant());
+        return dataPoints.get(dataPointIndex) == null ? "" : DATE_FORMATTER.format(dataPoints.get(dataPointIndex).getInstant());
```

Comment:
Actually, the issue #794 is about getting an `ArrayIndexOutOfBoundsException` rather than `NullPointerException` which is caused when `dataPoints` does not contain an index `dataPointIndex` - so the exact same thing still happens with the code above.

The proper fix would be:
```
if(dataPointIndex >= dataPoints.size()) {
  return "";
}
return dataPoints.get(dataPointIndex);
```

Please adjust the issue title and commit message as well.